src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,87 @@
+package main.java.duke;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
```

Comment:
Separation of package statements as per [guidelines](https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements)

```suggestion
package main.java.duke;

import java.io.File;
import java.io.FileWriter;
import java.io.IOException;

import java.nio.file.Paths;

import java.util.ArrayList;
import java.util.Scanner;
```